README.md

```diff
@@ -1,54 +1,82 @@
 # IBM Cloud SDK for Power Cloud
-To be updated
-power-go-client provides the Go implementation for operating the IBM Power Cloud platform to build LPARS .
-Installing
 
-Install the SDK using the following command
+The power-go-client project provides the Go SDK for IBM® Power Systems™ Virtual Server.
 
-```shell
-go get github.com/IBM-Cloud/power-go-client
-```
+## Prerequisites
+
+- An [IBM Cloud Account](https://cloud.ibm.com/registration)
+- An IBM Cloud [IAM API key](https://cloud.ibm.com/docs/account?topic=account-userapikey) or [Token](https://cloud.ibm.com/docs/account?topic=account-iamtoken_from_apikey) allow the SDK to access your account.
+- Go version 1.17 or above.
+
+## Install
+
+Install the SDK using the following methods.
 
-Update the SDK to the latest version using the following command
+### `go get` command
 
-```shell
+Use this command to download and install the SDK to allow your Go application to use it:
+```
 go get -u github.com/IBM-Cloud/power-go-client
 ```
 
-Using the SDK
+### Go modules
+
+If your application is using Go modules, you can add a suitable import to your Go application, like this:
+```go
+import (
+  "github.com/IBM-Cloud/power-go-client"
+)
+```
+then run `go mod tidy` to download and install the new dependency and update your Go application's
+`go.mod` file.
 
-You must have a working IBM Cloud account to use the APIs. Sign up if you don't have one.
+### `dep` dependency manager
 
-The SDK has examples folder which cites few examples on how to use the SDK. First you need to create a session.
+If your application is using the `dep` dependency management tool, you can add a dependency
+to your `Gopkg.toml` file. Here is an example:
+```
+[[constraint]]
+  name = "github.com/IBM-Cloud/power-go-client"
+  version = "1.1.0"
+
+```
+then run `dep ensure`.
+
+## Usage
+
+The SDK has [examples](examples) directory that shows how to use the SDK. First you need to create a session and use it for creating the client.
```

Comment:
Maybe the phrasing could be a little more clearer using this:
The SDK has `examples/<function>` directory . . .

Reply:
I have moved this statement to the end of the section. Please check again.